admin_manual/configuration_server/performance_tuning.rst

```diff
@@ -13,6 +13,26 @@ deployment.
 
 This chapter gives a few hands-on tips on how to achieve this.
 
+***************************
+PHP Version and Information
+***************************
+
+You will need to know your PHP version and configurations. To do this, create a 
+plain-text file named **phpinfo.php** and place it in your Web root, for 
+example ``/var/www/html/phpinfo.php``. (Your Web root may be in a different 
+location; your Linux distribution documentation will tell you where.) This file 
+contains just this line::
+
+ <?php phpinfo(); ?>
+
+Open this file in a Web browser:
+
+.. figure:: ../images/phpinfo.png
+
+Your PHP version is at the top, and the rest of the page contains abundant 
+essential system information such as active modules, active `.ini` files, and 
+much more.
```

Comment:
Maybe also note that this should be removed after debugging a problem as such a phpinfo.php contains a lot of sensitive data about the environment?